src/components/Login/Logout.jsx

```diff
@@ -27,7 +28,22 @@ const Logout = () => {
         <label id="labelLogout" htmlFor="btnLogout">
           Click the following logout button to log out of your pod:{' '}
         </label>
-        <LogoutButton onLogout={handleLogout} />
+        <button type="submit" onClick={() => setShowConfirmation(true)}>
+          Logout
+        </button>
+        {showConfirmation ? (
+          <dialog open>
+            <p>Do you want to log out now?</p>
+            <div>
+              <LogoutButton onLogout={handleLogout}>
+                <button>Logout</button>
+              </LogoutButton>
```

Comment:
From Line 38 to 40, we can use the default LogoutButton component for now. So instead, we can just use:

`<LogoutButton onLogout={handleLogout} />`

Reply:
That makes sense -- this has been updated.